modules/search/client/components/UsersList.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Avatar = ({ avatarSize, avatarSource, displayName, profileUrl }) => (
+  <div data-size={avatarSize}>
+    <a href={profileUrl}
+      aria-label={`Open user profile for ${displayName}`}>
+      <img className={`avatar avatar-${avatarSize} avatar-${avatarSource}`}
+        alt=''
+        // {alt={`Profile picture for ${userData.displayName}`}}
+        aria-hidden="true"
+        src={avatarSource}
+        href={profileUrl}
+        draggable="false" />
+    </a>
+  </div>
+);
+
+Avatar.defaultProps = {
+  avatarSize: 128
+};
+
+Avatar.propTypes = {
+  avatarSize: PropTypes.number,
+  avatarSource: PropTypes.string,
+  displayName: PropTypes.string,
+  profileUrl: PropTypes.string
+};
+
+const User = ({ userData }) => {
```

Comment:
Would be nice to extract this to an external reusable component (we could reuse it e.g. for the contacts list). That's okay for another PR too.

Reply:
Done in https://github.com/Trustroots/trustroots/pull/1009